frontend/src/components/ShootMessageDetails.vue

```diff
@@ -136,6 +136,17 @@ export default {
     canLinkToSecret () {
       return this.secretBindingName && this.namespace
     }
+  },
+  methods: {
+    icon (userError, temporaryError) {
+      if (userError) {
+        return 'mdi-account-alert'
+      }
+      if (temporaryError) {
+        return 'mdi-clock-alert'
+      }
```

Comment:
do you think we really need an own icon for temporary errors?

Reply:
I don't know. I thought has we highlight errors with known error codes, this could improve the ability to find out what is going on with your clusters faster. We can discuss this. As we also use this icon as an indication for cluster expiration it could irritate some users.

Reply:
I added a screenshot that shows the icon

Reply:
I removed the dedicated icon